nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -155,9 +155,9 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         try {
             session.exportTo(flowFile, response.getOutputStream());
             response.flushBuffer();
-        } catch (final IOException ioe) {
+        } catch (final Exception e) {
```

Comment:
Thinking through this a bit more, catching Exception is probably appropriate. Typically you want to not blanket catch when creating framework level code but this needs to be hardened to anything that may happen when responding. 

I am good with how it is now.